src/lib/utils/SynapseClient.tsx

```diff
@@ -113,7 +133,7 @@ export const getQueryTableResultsFromJobId = (
   sessionToken: string | undefined = undefined,
   endpoint: string = DEFAULT_ENDPOINT
 ): Promise<QueryResultBundle> => {
-  return doGet(`/repo/v1/entity/${entityId}/table/query/async/get/${jobId}`, sessionToken, endpoint)
+  return doGet(`/repo/v1/entity/${entityId}/table/query/async/get/${jobId}`, sessionToken, undefined, endpoint)
```

Comment:
Shouldn't all the API calls receive a new parameter `initCredentials` that gets passed down? Otherwise it never gets set?

Reply:
only session cookie calls need the include credentials